src/web/components/StickyBottomBanner/ReaderRevenueBanner.tsx

```diff
@@ -147,6 +153,9 @@ export const canShow = async ({
         .then((response) => response.json())
         .then((json) => {
             if (!json.data) {
```

Comment:
`data` meaning "should show banner"?

Reply:
yes, if `data` is missing in the response then there is no banner to show

Reply:
I know that's not even your change, but we could  const that up to make it clearer